custom-namespaces.yaml

```
@@ -162,6 +162,10 @@ contextual_services:
     - rburwei@mozilla.com
   spoke: looker-spoke-private
   views:
+    final_adM_forecasts:
```

nit. usually we just use lowercase identifiers
```suggestion
    final_adm_forecasts:
```